server/server.js

```diff
@@ -291,26 +291,27 @@ app.get('/logout', (req, res) => {
 
 app.post('/flic_button_press', Validator.header(['button-serial-number']).exists(), async (req, res) => {
   try {
-    const validationErrors = Validator.validationResult(req)
+    const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
 
     if (validationErrors.isEmpty()) {
       const serialNumber = req.get('button-serial-number')
       const batteryLevel = req.get('button-battery-level')
       const buttonName = req.get('button-name')
-      const apiKey = req.query.apikey
+      const alertApiKey = req.query.apikey
 
       // Log the vaiditiy of the API key
       // TODO (CU-gwxnde) Replace this with a 401 unauthorized if invalid
-      if (apiKey !== helpers.getEnvVar('FLIC_BUTTON_PRESS_API_KEY')) {
+      if (alertApiKey !== helpers.getEnvVar('FLIC_BUTTON_PRESS_API_KEY')) {
```

Comment:
Isn't this the other API key? not the BraveAlert one?

Reply:
:laughing: Yup, totally is. I'll fix that.

Reply:
Fixed!